fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -2399,6 +2402,13 @@ public long loadGlobalVariable(DataInputStream in, long checksum) throws IOExcep
         return checksum;
     }
 
+    // outline persistence
+    public long loadOutline(DataInputStream in, long checksum) throws IOException, DdlException {
+        VariableMgr.read(in);
```

Comment:
why not outlineMgr? and what's the difference between this method and loadOutlines